requirements/edx/kernel.in

```diff
@@ -116,7 +116,8 @@ openedx-calc                        # Library supporting mathematical calculatio
 openedx-django-require
 openedx-events>=8.3.0               # Open edX Events from Hooks Extension Framework (OEP-50)
 openedx-filters                     # Open edX Filters from Hooks Extension Framework (OEP-50)
-openedx-learning                    # Open edX Learning core (experimental)
+# openedx-learning                    # Open edX Learning core (experimental)
+openedx-learning @ git+https://github.com/openedx/openedx-learning@main  # Remove this once openedx-learning is tagged
```

Comment:
@bradenmacdonald could you please create a tag to deploy the new version of [openedx-learning](https://github.com/openedx/openedx-learning/blob/main/openedx_learning/__init__.py)?

Reply:
Sure, done. We'll need to merge this PR soon because v0.1.6 has a breaking change:

```
   File "/openedx/edx-platform/openedx/features/content_tagging/rules.py", line 99, in <module>
     def can_change_object_tag(user: User, object_tag: oel_tagging.ObjectTag = None) -> bool:
 AttributeError: module 'openedx_tagging.core.tagging.rules' has no attribute 'ObjectTag'
```

Reply:
I'm pinning the version to avoid unwanted updates while openedx-learning is in development/unstable.
Does that make sense? 1822ed9d3c63a446a5bab42ce0185f22623644cf

Reply:
Yes that makes sense.